src/ui/src/js/services/token_service.js

```diff
@@ -11,14 +11,14 @@ tokenService.$inject = ["$http", "localStorageService"];
 export default function tokenService($http, localStorageService) {
   const service = {
     getToken: () => {
-      return localStorageService.get("token", "sessionStorage");
+      return localStorageService.get("token");
```

Comment:
There didn't seem to be a way to gracefully handle multiple tabs / windows while storing the token in session storage.  Moving it to the local storage has the effect of all tabs / windows essentially sharing the same login session.